deploy/crds/ssp.kubevirt.io_ssps_crd.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Is the name of this crd correct?

Reply:
The name was generated from the 'kind' and 'group', but we can change the name. Or change the 'kind' too, I was not sure what would be a good kind.

Reply:
Yes, the name is correct